build.gradle

```diff
@@ -96,12 +96,16 @@ task runExport(dependsOn: ['classes'], type: JavaExec) {
 		systemProperty("databasePassword", databasePassword)
 		systemProperty("environment", "export")
 
-		def argumentsSet = project.hasProperty('dataExportSpecFile') && project.hasProperty('outputFile');
+		def argumentsSet = (project.hasProperty('dataExportSpecFile') || project.hasProperty('dataExportSpec')) && project.hasProperty('outputFile');
```

Comment:
@arya-hemanshu do you think this will be used by the user directly from the terminal? If not I would suggest to create a new task for it.

Reply:
Or separate the two cases somehow to not confuse with many properties.

Reply:
The only reason i don't want to do it, is because there will be a lot of repetition of code, as then we need to create a new Runner class, though the only difference would be of one argument and if we are keeping the same Runner class, then having the same task name is what i would prefer, as this is just an additional ```if``` statement. I also feel it gives consistency in how tasks are run, as no matter what the way is to run the recipe, the gradle task is always ```runExport```

Reply:
I see. Then I'll push up the bidding :) and suggest to call it `recipe` and accept both in the same property. The user will run it from the command line with a file but in the `run` method we can differentiate between the file or recipe. If you don't find this to your liking either please proceed with merging.

Reply:
@lorenaqendro  i think it would be good if we push it like that, as differentiating between a path and a recipe could create some issues, e.g. this is the way i think it would happen, both will come as string, and in order to see if this a path or recipe, we need to parse it to file and if that raises an exception, we will pass it as direct recipe, but the problem arises when user has genuinely made error in the path and same could happen with recipe.
Please let me know if you have any other approach in mind, happy to make the changes that way :)